concent_api/concent_api/tests/test_check_payment_interface_chain.py

```diff
@@ -0,0 +1,46 @@
+from assertpy import assert_that
+import pytest
+
+from django.conf import settings
+from django.test import override_settings
+
+from golem_sci import chains
+
+from concent_api.system_check import check_payment_interface_chain
+from concent_api.system_check import create_error_59_payment_interface_chain_is_not_set
+from concent_api.system_check import create_error_60_payment_interface_chain_is_not_a_string
+from concent_api.system_check import create_error_61_payment_interface_chain_is_invalid
+
+
+# pylint: disable=no-self-use
+class TestPaymentInterfaceChainCheck():
+
+    def test_that_payment_interface_chain_check_will_produce_error_when_payment_backend_is_not_set(self):
+        del settings.PAYMENT_INTERFACE_CHAIN
+        errors = check_payment_interface_chain()
+
+        assert_that(len(errors)).is_equal_to(1)
+        assert_that(errors).is_equal_to([create_error_59_payment_interface_chain_is_not_set()])
+
+    @pytest.mark.parametrize(('chain', 'error'), [
+        (chains.RINKEBY, []),
+        (chains.MAINNET, [])
+    ])
+    def test_that_payment_interface_chain_from_golem_sci_will_not_produce_error(self, chain, error):
+        with override_settings(PAYMENT_INTERFACE_CHAIN=chain):
+            errors = check_payment_interface_chain()
+            assert_that(errors).is_equal_to(error)
```

Comment:
My suggestion would be to check, that error list is empty in case of correctly set values:
```suggestion
            assert_that(errors).is_empty()
```
It would also require less parameters (just `chain`).

Reply:
Fixed :)